src/components/app.js

```diff
@@ -41,10 +42,51 @@ const getMuiTheme = (theme) => {
     }
 };
 
-const App = () => {
+const SignInCallback = () => {
+    const history = useHistory();
+
+    function handleCallback() {
+        new UserManagerHelper().getUserManagerInstance()
```

Comment:
Can you simplify the design to avoid having to "new" an object here ?

Reply:
This class UserManagerHelper is not a helper. In addition to the bad naming i think this design is not best way to do it in javascript.